### DISCUSSION TITLE
Github icon in repository meaning

### DISCUSSION THREAD

**Discussion by 0xtejas** [community/community#23562] [New to GitHub]
<p><img src="https://user-images.githubusercontent.com/47889755/181077912-4a44f001-8b68-45ee-bb3e-191112898cf4.png" alt="image" data-base62-sha1="uajrofMZV9KuIeY3YS7D5vHae3s" width="439" height="182"><br>
Not sure what this icon means, I tried adding a submodule. But when I clicked that, it doesn’t redirect me to the module</p>
<p>Can some one tell me what the icon represents?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>yes, it’s a submodule but maybe the setup is incorrect,</p>
<p>check this out,</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://blog.bitsrc.io/how-to-utilize-submodules-within-git-repos-5dfdd1c62d09">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/47092464/181077914-71aa8df1-18d3-4069-880c-ed5750a8c093.png" class="site-icon" width="256" height="256">

      <a href="https://blog.bitsrc.io/how-to-utilize-submodules-within-git-repos-5dfdd1c62d09" target="_blank" rel="noopener" title="02:52PM - 29 August 2021">Medium – 29 Aug 21</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/459;"><img src="https://user-images.githubusercontent.com/47092464/181077915-33b6c0fb-5826-4ad8-b471-9a70de14ecc4.jpeg" class="thumbnail" width="690" height="459" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/e/aeccf7f9104a5bff7430b31847f5b31ff37d894d_2_690x459.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/a/e/aeccf7f9104a5bff7430b31847f5b31ff37d894d_2_1035x688.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/a/e/aeccf7f9104a5bff7430b31847f5b31ff37d894d.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/e/aeccf7f9104a5bff7430b31847f5b31ff37d894d_2_10x10.png"></div>

<h3><a href="https://blog.bitsrc.io/how-to-utilize-submodules-within-git-repos-5dfdd1c62d09" target="_blank" rel="noopener">How to Utilize Submodules within Git Repos</a></h3>

  <p>One Solution When the Primary Code Can be Open Source, but Specific Content Needs to be Private</p>

  <p>
    <span class="label1">Reading time: 12 min read</span>
  </p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
